src/main/webapp/style.css

```diff
@@ -5,3 +5,33 @@ h3 {
 .srp-container {
   display: flex;
 }
+
+#auth {
+  width: 47px;
```

Comment:
I didn't check, but this feels like you're hardcoding the height and width to fit the size of the text on your browser on your OS on your device. Why not just let it be the size that it renders at?